package.json

```diff
@@ -190,6 +190,7 @@
     "node-polyfill-webpack-plugin": "1.1.4",
     "prettier": "2.3.2",
     "prettier-config-constellation": "0.0.5",
+    "react-circular-progressbar": "2.0.4",
```

Comment:
I think this should go in the `package/blockchain-wallet-v4-frontend/package.json` instead of this one and in the dependencies instead of devDependencies.